pkg/util/util_test.go

```diff
@@ -73,9 +73,12 @@ func TestMaskPassword(t *testing.T) {
 		"pass='foobar' host='localhost'":                                     "pass='foobar' host='localhost'",
 		"host='localhost' password='foobar'":                                 "host='localhost' password='****'",
 		"password:foobar  host: localhost":                                   "password:****  host: localhost",
-		"password:foo bar  host: localhost":                                  "password:**** bar  host: localhost",
+		"password:foo bar  host: localhost":                                  "password:****  host: localhost",
```

Comment:
Can we have more cases like `Password: "foo bar"`, `password="foo bar"`, `Password="foo bar"` ?